doc/man/salt.7

```diff
@@ -105534,7 +105534,7 @@ Connection module for Amazon EC2
 T}
```

Comment:
Do not edit the salt.7 file, this is autogenerated.

Reply:
reverted.